requirements.txt

```diff
@@ -2,3 +2,4 @@ numpy
 scipy
 pytest
 pytest-cov
+python_version >= "3.6"
```

Comment:
I haven't seen this before, but won't this make pip install [python_version](https://pypi.org/project/python-version/) when you run `pip install -r requirements.txt`?

Reply:
My bad. I assumed it forces pip to make sure that the Python version used for installation is as specified, but actually this does a different thing - such syntax allows to install certain dependecies conditionally on the Python version (decribed here https://peps.python.org/pep-0508/#environment-markers). So this needs to be removed actually. But is there a way to enforce a specific python version in the `requirements.txt`?

Reply:
I don't think we can specify the Python version in `requirements.txt`, no but I think we can set it in `setup.cfg` or `setup.py` since we use Setuptools.